spoti-vote/src/components/Sidebar/Infos.jsx

```diff
@@ -41,28 +41,56 @@ class Infos extends Component {
 	constructor() {
 		super();
 		this.state = {
-			playlistData: {
-				playlists: []
-			}
+			playlists: []
+		}
+	}
 
+	componentDidUpdate() {
+		//This is only run, when the update request reports a different amount of playlists then the current numbers of playlists
+		//It will fetch all the playlists again. (Do we need this)
+		if (this.props.loggedIn === true && this.props.numPlaylists !== this.state.playlists.length) {
+			fetch('http://localhost:8888/instance/playlists?id='+window.location.pathname.split('/')[2], {
+		    }).then((response) => response.json().then(data => {
+				if (this.state.playlists.length !== data.length) {
+					this.setState({
+						playlists: data
+					});
+				}
+		    })).catch(function() {
+				window.location.reload();
+			});
 		}
 	}
 
-	componentDidMount() {
-		let access_token = this.props.token;
+	render() {
```

Comment:
note that render function is invoked every time a state or prop updates. logic should not be too intense